src/store/reducers/registrationCommon.ts

```diff
@@ -41,16 +41,17 @@ export const thunkAuthSignin = createAsyncThunk(
   'auth/signin',
   async (options: Signin, { rejectWithValue }) => {
     try {
+      console.log(options);
       const res = await fetchSignIn(options);
-
+      console.log(res);
       if (res.status !== 201) {
         const err: { message: string; statusCode: number } = await res.json();
 
         throw new Error(`${err.message}`);
       }
       return await res.json();
     } catch (error) {
-      console.log('errror', error);
+      console.log('error', error);
 
       return rejectWithValue(getErrorMessage(error));
     }
```

Comment:
You have async await syntax with try catch. Catch is needed for catching every errors
Why we need trhrow new here?
https://javascript.info/async-await
![image](https://github.com/kosta4310/farmers-market/assets/86911989/d75b446e-c1a3-4bc1-a07e-d6197d6a5db5)

Reply:
Agree, but it's not my code. we don't need to throw new error or even alert I would say